openfisca_us/parameters/tax/ALD/dependents/child_c.yaml

```diff
@@ -0,0 +1,6 @@
+description: "The weighted average of childcare costs in the US. 7165 is the weighted average from the Child Care in America: 2016 State Fact Sheets."
+values:
+  2013-01-01: 0
+metadata:
+  unit: currency-USD
+  reference: http://taxcalc.pslmodels.org/guide/policy_params.html#ald-dependents-child-c
```

Comment:
I haven't myself been consistent on this, but should we start enforcing title/url pairs for references? Seems like it could help later on with serving to UIs

Reply:
Done